flake8_quotes/__init__.py

```diff
@@ -54,10 +54,12 @@ def _register_opt(parser, *args, **kwargs):
     @classmethod
     def add_options(cls, parser):
         cls._register_opt(parser, '--quotes', action='store',
-                          parse_from_config=True,
+                          parse_from_config=True, type='choice',
+                          choices=list(cls.INLINE_QUOTES),
```

Comment:
As a programmer, I don't exactly know what happens when I convert a dictionary to a list; do I get the keys, do I get the values. Can we move to `cls.INLINE_QUOTES.keys()` here (or `sorted(cls.INLINE_QUOTES.keys())` if you want to guarantee consistent ordering as you mentioned in #33)